draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1044,686 @@ uCDN-A                         dCDN-B                         dCDN-C
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Array of GenericTriggerExtension objects (see <xref target="generic-extension-object"/>).
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
+   Mandatory: No. The default is no extensions.
    </t>
 
-   <t hangText="Name: desired-status">
-   <vspace blankLines="1"/>
-   Description: The target status for an existing trigger as defined in <xref target="trigger-status"/>
+   <t hangText="Name: labels">
    <vspace blankLines="1"/>
-   Value: JSON string. 
+   Description: Array of trigger labels.
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
-   </t>
-
-   </list></t>
-   </list></t>
-
-   </section>
+   Value: Array of Trigger labels, Each label is encoded as a JSON string and SHOULD be unique per trigger.
 
 
-   <section title="Trigger Status Resources" anchor="trigger-status-resources"><t>
-    Trigger Status Resources MUST use a MIME media type of
-   "application/cdni; ptype=ci-trigger-status.v2".</t>
-
-   <t>
-   A Trigger Status Resource is encoded as a JSON object containing the
-   following name/value pairs:</t>
+   <vspace blankLines="1"/>
+   Mandatory: No. The default is no labels.
+   </t>
 
-   <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3"><t hangText="Name: trigger">
+   <t hangText="Name: cdn-path">
    <vspace blankLines="1"/>
-   Description: The Trigger Specification POSTed in the body of
-         the CI/T Command. Note that this need not be a byte-for-byte
-         copy. For example, in the JSON representation dCDN may
-         re-serialize the information differently.
+   Description: The chain of CDN PIDs of CDNs that have already created this trigger resource. 
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Non-empty array of JSON strings, where each string is a CDN PID as defined in <xref target="cdn-pid"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: No. The default is no CDN path.
    </t>
 
    <t hangText="Name: ctime">
    <vspace blankLines="1"/>
-   Description: The time at which the CI/T Command was received by the
-         dCDN. The time is determined by dCDN; there is no requirement
-         to synchronize clocks between interconnected CDNs.
+   Description: The time at which the trigger resource was received by the dCDN. 
+   The time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: Absolute Time, as defined in <xref target="absolute-time"/>.
+   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
    <vspace blankLines="1"/>
    Mandatory: Yes.
```

Comment:
This comment applies to ctime, mtime, and etime.  Prior to the RESTful changes, it was clear that these should only appear in the Trigger Status resources.  Now that we're making the RESTful changes, should we indicate that this is only mandatory for the dCDN for extra clarity?  The language leaves this ambiguous (although noting that this is determined by the dCDN), although the examples certainly depict the correct behavior.

Reply:
This is one change I didn't get to, clearly indicating which trigger attributes are mandatory for each party. 
Will add it now